right/src/keyboard_layout.h

```diff
@@ -33,6 +33,32 @@ typedef enum {
     UHK_KEY_LPRESSLAYER,
 } uhk_key_type_t;
 
+enum {
```

Comment:
Well, directional mouse movement *is* a practical thing, I've seen a lot of people use it, and until I had mouse keys on the ErgoDox, I used them too, and will use them on the Keyboardio too. On the UHK, I'd use `Mouse+Q`, `Mouse+E`, `Mouse+Z`, and `Mouse+C` for directional movement, and `Mouse+C/V/B` for buttons. Been doing that during testing, and is - for me - much more convenient than `Mouse+W` + `Mouse+A`. The less keys I need to hold, the better.

So I'd argue strongly that the Agent should be improved, instead of the firmware dumbed. Or even if the Agent is not changed, having the option in the firmware is still useful, because then one can recompile the firmware with their own layout, that can make use of these things.

The other things are less useful when combined this way (especially scrolling; I could see a use for simultaneous button pressing with one key), but this layout makes it more consistent. I can be convinced to turn non-movement things into distinct values instead of bitfields, but movements is something I very strongly feel should be a bitfield.